book/03_Geospatial_data_files/01_Geospatial_data_formats.md

```diff
@@ -0,0 +1,53 @@
+# Geospatial data formats
+
+## Vector data formats
+
+In this tutorial we will be using the following vector data formats:
+
+### 1. SHP
+
+Shapefile is the most widely known format for distributing geospatial data. It is a standard first developed by ESRI almost 30 years ago, which is considered ancient in software development.^3^ 
+
+The Shapefile consists of several files: in addition to one file with the actual geometry data, another file for defining the coordinate reference system is needed, as well as a file for defining the attributes and a file to index the geometries. This makes operating Shapefiles slightly clunky and confusing. However, Shapefile has been around for so long that any GIS software supports handling it.^3^
+
+Internally, Shapefile uses Well-known binary (WKB) for encoding the geometries. This is a compact format that is based on tabular thinking, i.e. the row and column number of a value is significant. A minor nuisance is the limitation of the attribute field names to 10 characters and poor Unicode support, so some abbreviations and forcing to ASCII may have to be used.^3^
+
+For more details about the shapefiles you can refer to the [ESRI Shapefile Technical Description](https://www.esri.com/content/dam/esrisites/sitecore-archive/Files/Pdfs/library/whitepapers/pdfs/shapefile.pdf)
+
+### 2. GeoJSON
```

Comment:
Is it possible to have an illustration of some kind here? Maybe a screenshot or the first five lines of , e.g., https://gist.githubusercontent.com/wavded/1200773/raw/e122cf709898c09758aecfef349964a8d73a83f3/sample.json)?